packages/usa-icon/src/test/icon.spec.js

```diff
@@ -0,0 +1,44 @@
+const assert = require("assert");
```

Comment:
Is assert needed as a dev dependency?

Reply:
Added only gulp devDependencies to the package since the other dependencies (including assert) were included with node.